.github/workflows/deploy-github-pages.yml

```diff
@@ -0,0 +1,78 @@
+name: Deploy Hugo site to Pages
+
+on:
+  push:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      HUGO_VERSION: 0.137.1
```

Comment:
Do we need to hardcode this version here?  Or is there a way of defining this value in the equivalent of a `requirements.txt`?